pkg/fqdn/cache.go

```diff
@@ -137,25 +132,30 @@ type DNSCache struct {
 
 	// perHostLimit is the number of maximum number of IP per host.
 	perHostLimit int
+
+	// minTTL is the minimun TTL value that a cache entry can have, if the TTL
+	// sent in the Update is lower, the TTL will be owerwritten to this value
+	minTTL int
```

Comment:
Could you also leave a comment here saying minTTL and perHostLImit are not protected by the mutex. Perhaps also move them above the mutex since our convention is that everything below is protected.